src/include/wiredtiger.h.in

```diff
@@ -5522,6 +5522,11 @@ struct __wt_storage_source {
  */
 #define WT_PAGE_LOG_LSN_MAX UINT64_MAX
 
+/*!
+ * LSN used by Server during node step up process to avoid evicting pages.
+ */
+#define WT_DISAGG_START_LSN 4294967296
```

Comment:
nit: Can you please write it as `(1LL<<32)`. It'd make it more clear what this constant is.